arrow/locales.py

```diff
@@ -850,13 +920,19 @@ class FinnishLocale(Locale):
 
     day_abbreviations = ["", "ma", "ti", "ke", "to", "pe", "la", "su"]
 
-    def _format_timeframe(self, timeframe, delta):
+    # TODO: Fix return type
+    def _format_timeframe(self, timeframe: TimeFrames, delta: Union[float, int]) -> Tuple[str, str]:  # type: ignore
         return (
             self.timeframes[timeframe][0].format(abs(delta)),
             self.timeframes[timeframe][1].format(abs(delta)),
         )
```

Comment:
The return type of this particular method is not match with others. Should I change return type of `_format_timeframe()` and signature of all related methods?

Reply:
Let's make all of the `_format_timeframe` functions consistent. This might be related to https://github.com/arrow-py/arrow/issues/789.

Reply:
I think the issue can handle in separate PR. Can I handle on another PR?

Reply:
What you mean is we have to change return type of all `_format_timeframe()`? Then we also have to modify logic of `describe()` and `describe_multi()`. but I think I can handle it because I'm not that aware of locale things.. Can you help this? @jadchaar

Reply:
@jadchaar  If I change return type of `_format_timeframe` to `Union[str, Tuple[str, str]]`, mypy returns

```
arrow/locales.py:143: error: Argument 1 to "_format_relative" of "Locale" has incompatible type "Union[Tuple[str, str], str]"; expected "str"
arrow/locales.py:145: error: Incompatible return value type (got "Union[Tuple[str, str], str]", expected "str")
arrow/locales.py:163: error: Argument 1 to "join" of "str" has incompatible type "List[Union[Tuple[str, str], str]]"; expected "Iterable[str]"
```

I think just add `type: ignore` on this PR and refactor locale related code using strategy pattern or something in another PR. The refactoring must be quite big I think.